compiler-core/src/type_/environment.rs

```diff
@@ -491,18 +491,22 @@ impl<'a> Environment<'a> {
         }
     }
 
-    /// Converts entities with a usage count of 0 to warnings
-    pub fn convert_unused_to_warnings(&mut self) {
+    /// Converts entities with a usage count of 0 to warnings.
+    /// Returns the list of location for the removed unused lsp action.
```

Comment:
Looks like it's specifically unused imported modules, can we say that here please